packages/terra-date-picker/src/DatePicker.jsx

```diff
@@ -0,0 +1,182 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import ReactDatePicker from 'react-datepicker';
+import moment from 'moment';
+import ResponsiveElement from 'terra-responsive-element';
+import DateInput from '../lib/DateInput';
+import '../lib/DatePicker.scss';
```

Comment:
I think all these imports should be relative:
```
import DateInput from './DateInput';
import './DatePicker.scss';
```
These paths will work from src and lib